hazelcast-jet-core/src/main/java/com/hazelcast/jet/config/JobConfig.java

```diff
@@ -1154,4 +1154,15 @@ public int hashCode() {
                 serializerConfigs, classLoaderFactory, initialSnapshotName
         );
     }
+
+	@Override
+	public String toString() {
+		return "JobConfig [name=" + name + ", processingGuarantee=" + processingGuarantee + ", snapshotIntervalMillis="
```

Comment:
Thanks for the PR! Only one minor comment:

* should be `JobConfig{}` as that's the convention generally used

Reply:
Thanks for the review. I have updated the bracket-style to '{}'. Please let me know if any other changes need to be done.